schemas/objects/bottomBanner.js

```diff
@@ -20,5 +20,16 @@ export default {
       type: 'button',
       validation: Rule => Rule.required(),
     },
+    {
+      title: 'Second Button',
+      name: 'secondButton',
+      type: 'button',
+      description: '(Optional)'
+    },
+    {
+      title: 'Image',
+      name: 'image',
+      type: 'customImage',
```

Comment:
This is optional too, I guess we should be consistent when adding the description "(Optional)", just don't have it at all or add it everywhere it's needed

Reply:
I will include the (optional) description. I think this will be helpful to the content editor.